version/version.go

```diff
@@ -91,3 +84,17 @@ func isValidVersion(version string) bool {
 	}
 	return true
 }
+
+// printServerVersion outputs current server version
+func printServerVersion(client *houston.Client, out io.Writer) error {
```

Comment:
why do we need to copy this?
DRY? for https://github.com/astronomer/astro-cli/blob/e7a3f35b98584e3685e9a8f9764a23f4b5c17c71/version/version.go#L33-L40

Reply:
Yeah, I was trying to keep it DRY.  This method is now being invoked twice.  Once on line `33` and once on `69`.